client/src/containers/projects/custom-project/cost-details/table/index.tsx

```diff
@@ -28,12 +32,46 @@ export interface CostItem {
 }
 
 interface CostDetailTableProps {
-  data: CostItem[];
+  data: ReturnType<typeof useCustomProjectOutput>["costDetailsProps"];
 }
+
 const CostDetailTable: FC<CostDetailTableProps> = ({ data }) => {
   const [sorting, setSorting] = useState<SortingState>([]);
+
+  const higherNegative = Math.max(
+    ...data.map(({ sensitivityAnalysis }) =>
+      Math.abs(sensitivityAnalysis?.changePctLower ?? 0),
+    ),
+  );
+  const higherPositive = Math.max(
+    ...data.map(({ sensitivityAnalysis }) =>
+      Math.abs(sensitivityAnalysis?.changePctHigher ?? 0),
+    ),
+  );
```

Comment:
Nit: I think you can put these in the useMemo hook, if they are only part of the parseData calculation